12주차/(P)17686/17686(P)_python_donghoon.py

```diff
@@ -0,0 +1,9 @@
+import re
+def solution(files):
+    return [''.join(x) for x in sorted([re.split("("+re.match(r'^[^\d]*(\d{1,5})', s).group(1)+")", s) for s in files], key=lambda x: (x[0].lower(), int(x[1])))]

```

Comment:
ps에선 크게 상관없는 (아니면 오히려 숏코딩 취미로 더욱 지향되는) 부분이긴 하지만, 깃 코드리뷰에선 한 라인에 코드가 긴 것을 지양하고 있어요. 같은 라인에 여러 코멘트를 달게 되기 때문이에요 ㅜㅜ 제출할 때엔 1줄로 하고, git에 올릴 땐 아래와 같이 올려도 좋아보여요 😄 (임의로 개행 분리한거라 더 좋은 컨벤션이 있을 것 같아요)
```suggestion
    return [''.join(x) for x in sorted(
        [re.split(
            "("+re.match(r'^[^\d]*(\d{1,5})', s).group(1)+")", s
        ) for s in files], 
        key=lambda x: (x[0].lower(), int(x[1]))
    )]

```

Reply:
받아들이겠습니다! 깃허브 suggestion도 배워 가요 👍🏼